cpp/example_code/cross-service/serverless-aurora/README.md

```diff
@@ -0,0 +1,98 @@
+
+# Create an item tracker web app using Amazon Relational Database Service (Amazon RDS), Amazon Simple Email Service (Amazon SES), and the AWS SDK for C++.
```

Comment:
Create an item tracker web app using Amazon RDS, Amazon SES, and the AWS SDK for C++
(No need to use long service names or periods in titles.)